doc/source/changes/version_0_26.rst.inc

```diff
@@ -44,6 +44,27 @@ Miscellaneous improvements
 * replaced `na` argument of `read_csv`, `read_excel`, `read_hdf` and `read_sas` functions by `fill_value`
   (closes :issue:`394`)
 
+* allowed to pass a label or a named group as `sheet_name` argument of the method `to_excel` and
+  `key` argument of the method `to_hdf` (closes :issue:`328`):
+
+    >>> arr = ndtest((4, 4, 4))
+
+    >>> # iterate over labels of a given axis
+    >>> for label in arr.a:
+    ...     arr[label].to_excel('my_file.xlsx', label)
```

Comment:
I have a problem with this example, because this is not the way users should dump an array to multiple sheets, because it opens & close the file several times, which is a bit costly. They should use open_excel in that case:
```python
with open_excel('my_file.xlsx') as wb:
    for label in arr.a:
        wb[label] = arr[label]
```
so I guess Workbook[Group] = xxx should be implemented too before the release. This can be done in this PR or another, as you prefer, or we might want to implement #423 directly.

Reply:
Indeed, wrong example. That makes me think we should explain this (not using `to_excel` in a loop) in the tutorial.

Reply:
OK to implement Workbook[Group] = xxx in this PR. Seems legit.